plutus-pab/app/Cli.hs

```diff
@@ -94,29 +96,47 @@ import           Plutus.PAB.Effects.Contract.ContractExe  (ContractExe)
 import           Plutus.PAB.Effects.Contract.ContractTest (TestContracts (Currency))
 import qualified Plutus.PAB.Monitoring.Monitoring         as LM
 import qualified Plutus.PAB.Simulator                     as Simulator
-import           Plutus.PAB.Types                         (Config (..), chainIndexConfig, metadataServerConfig,
-                                                           nodeServerConfig, requestProcessingConfig,
-                                                           walletServerConfig)
+import           Plutus.PAB.Types                         (Config (..), DbConfig (..), chainIndexConfig,
+                                                           metadataServerConfig, nodeServerConfig,
+                                                           requestProcessingConfig, walletServerConfig)
 import qualified Plutus.PAB.Webserver.Server              as PABServer
 import           Plutus.PAB.Webserver.Types               (ContractActivationArgs (..))
 import           Wallet.Emulator.Wallet                   (Wallet (..))
 
+runNoConfigCommand ::
+    Trace IO (LM.AppMsg ContractExe)  -- ^ PAB Tracer logging instance
+    -> NoConfigCommand
+    -> IO ()
+runNoConfigCommand trace = \case
```

Comment:
:nail_care: could you insert some comments here separating the individual commands? 

```hs
-- Migrate the database or create if it doesn't exist
    Migrate{dbPath} ->

```

Just to make the individual commands stand out more.